src/app/subscriptions/data/index.ts

```diff
@@ -26,7 +26,7 @@ export const DiscoverySubscriptionCollection = {
 }
 export const SubscriptionCollection = {
```

Comment:
I think it's worth remembering that there are multiple subscriptions with different info. This is KDSSubscription but there are also DiscoverySubscription that are shown in there: https://deploy-preview-2664--kuma-gui.netlify.app/gui/meshes/default/data-planes/protocol-gateway_delegated-0/overview

Reply:
Yes definitely, tbh I'm guessing this code here will change a little to make it cope with both types, I made a half thought out reference to it in the PR desc also

> I'm guessing we'll want the same layout for DPP subscriptions, I plan to do this as a follow up and there will some changes made on top of here to make the panel work in both places.

Its kinda in the back of my head, and doing this will be coming up next 👍